.github/workflows/linux-eic-shell.yml

```diff
@@ -1233,3 +1235,16 @@ jobs:
           issue-number: ${{ github.event.pull_request.number }}
           body-file: capybara_${{ github.event.pull_request.number }}.md
           edit-mode: replace
+
+  cancel-container:
+    if: cancelled() || failure()
+    needs:
+    - trigger-container
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Cancel container
+        run:
+          echo \
+          curl -X POST \
+             -F "token=${{ secrets.EICWEB_CONTAINER_TRIGGER }}" \
```

Comment:
Probably not the correct token. This probably only allows triggering, not canceling.

Reply:
```suggestion
             -F "token=${{ secrets.EICWEB_CONTAINER_CANCEL }}" \
```